triagebot.toml

```diff
@@ -236,6 +236,15 @@ trigger_files = [
     "src/tools/jsondoclint",
 ]
 
+[autolabel."A-attributes"]
+trigger_files = [
+    "compiler/rustc_codegen_ssa/src/codegen_attrs.rs",
+    "compiler/rustc_passes/src/check_attr.rs",
+    "compiler/rustc_attr_parsing",
+    "compiler/rustc_attr_data_structures",
+    "compiler/rustc_attr_validation",
```

Comment:
While this doesn't exist [yet](https://github.com/rust-lang/rust/pull/135726) on master, I'd say it's fine. We can always update `triagebot.toml` later if things change. I just hope triagebot won't freak out ^^' but I doubt that.